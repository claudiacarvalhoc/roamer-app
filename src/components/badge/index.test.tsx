import React from 'react';
import Badge, { BadgeProps } from './index';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

const getShallowWrapper = (props) => {
    return shallow(<Badge {...props} />);
};

describe('<Badge />', () => {
    it('render correctly when type is orange', () => {
        const props: BadgeProps = {
            className: 'test',
            name: 'android',
            type: 'orange',
        };
        const component = getShallowWrapper(props);
        expect(toJson(component)).toMatchSnapshot();
    });
    it('render correctly when type is green', () => {
        const props: BadgeProps = {
            className: 'test',
            name: 'android',
            type: 'green',
        };
        const component = getShallowWrapper(props);
        expect(toJson(component)).toMatchSnapshot();
    });
});
