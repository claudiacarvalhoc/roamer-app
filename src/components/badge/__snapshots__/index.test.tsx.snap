// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Badge /> render correctly when type is green 1`] = `
<div
  className="tag test green"
>
  android
</div>
`;

exports[`<Badge /> render correctly when type is orange 1`] = `
<div
  className="tag test orange"
>
  android
</div>
`;
